README.md

```diff
@@ -1,11 +1,11 @@
 SELinux Policy Cookbook
 ======================
-This cookbbok can be used to manage SELinux policies and components (rather than just enable / disable enforcing).  
+This cookbbok can be used to manage SELinux policies and components (rather than just enable / disable enforcing).
```

Comment:
EOL spaces are automatically removed in my editor. As far as I known, EOL spaces are also not the way to go, but extra blank lines should be used.
I'll revert those parts of my commit.